content/en/tracing/profiling/search_profiles.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Search Profiles
+kind: Documentation
+further_reading:
+    - link: 'tracing/profiling/trace_profile'
+      tag: 'Documentation'
+      text: 'Connect traces and profiles.'
+    - link: 'tracing/profiling/code_level_metrics'
+      tag: 'Documentation'
+      text: 'View code level metrics.'  
+    - link: 'tracing/profiling/getting_started'
+      tag: 'Documentation'
+      text: 'Enable continuous profiling for your application.'
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+{{< img src="tracing/profiling/search_profiles.gif" alt="Search profiles by tags">}}
```

Comment:
Might be worth not using current time so the cpu and alloc metrics in each row aren't missing, until we fix that :-)